doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_cli.rst

```diff
@@ -21,6 +21,151 @@ Persistent storage
+----------------------------------------------
+
+	Set a Light Lightness Control Server's current Mode without requesting a response.
+
+	* ``enable | disable``: Mode to set.
+
+
+``mdl_ctrl occupancy-get``
+--------------------------
+
+	Get a Light Lightness Control Server's current Occupancy Mode.
+
+
+``mdl_ctrl occupancy-set <enable | disable>``
+----------------------------------------------
+
+	Set a Light Lightness Control Server's current Occupancy Mode and wait for a response.
+
+	* ``enable | disable``: Occupancy Mode to set.
+
+
+``mdl_ctrl occupancy-set-unack <enable | disable>``
+----------------------------------------------------
+
+	Set a Light Lightness Control Server's current Occupancy Mode without requesting a response.
+
+	* ``enable | disable``: Occupancy Mode to set.
+
+
+``mdl_ctrl light-onoff-get``
+----------------------------
+
+	Get a Light Lightness Control Server's current OnOff state.
+
+
+``mdl_ctrl light-onoff-set <onoff> [transition_time_ms  [delay_ms]]``
+---------------------------------------------------------------------
+
+	Tell a Light Lightness Control Server to turn the light on or off and wait for a response.
+
+	* ``onoff``: OnOff state value to set. Use `on`, `enable`, or any non-zero value to enable the state.
+	* ``transition_time_ms``: If present, defines the transition time in the message in milliseconds.
+	* ``delay_ms``: If present, defines the delay in the message in milliseconds.
+
+
+``mdl_ctrl light-onoff-set-unack <onoff> [transition_time_ms  [delay_ms]]``
+---------------------------------------------------------------------------
+
+	Tell a Light Lightness Control Server to turn the light on or off without requesting a response.
+
+	* ``onoff``: OnOff state value to set. Use `on`, `enable`, or any non-zero value to enable the state.
+	* ``transition_time_ms``: If present, defines the transition time in the message in milliseconds.
+	* ``delay_ms``: If present, defines the delay in the message in milliseconds.
+
+
+``mdl_ctrl prop-get <id>``
+--------------------------
+
+	Get a Light Lightness Control Server property value.
+
+	* ``id``: ID of the property to get.
+
+
+``mdl_ctrl prop-set <id> <decimal> <frac>``
+-------------------------------------------
+
+	Set a Light Lightness Control Server property value and wait for a response.
+
+	* ``id``: ID of the property to set.
+	* ``decimal``: Decimal part of the property value.
+	* ``frac``: Fractional part of the property value.
+
+
+``mdl_ctrl prop-set-unack <id> <decimal> <frac>``
+-------------------------------------------------
+
+	Set a Light Lightness Control Server property value without requesting a response.
+
+	* ``id``: ID of the property to set.
+	* ``decimal``: Decimal part of the property value.
+	* ``frac``: Fractional part of the property value.
+
+
+``mdl_ctrl coeff-get <id>``
+---------------------------
+
+	Get a Light Lightness Control Server Regulator Coefficient value.
+
+	* ``id``: ID of the coefficient to get.
```

Comment:
Should we document which ids that is valid? I seem to remember that i needed to dig a little bit to find a valid id to test this with.